### PR TITLE
Add support for printing floating point numbers in XMM/YMM

### PR DESCRIPTION
**Pull request by torusrxxx**
Now only ST(...) is missing (ldconvert required)

### CODE REVIEW

File: src/dbg/stringformat.cpp [hunk 1]
```diff
@@ -21,12 +21,103 @@ enum class StringValueType
     FloatingPointDouble
 };
 
+template<class T> String printFloatValue(FormatValueType value)
+{
+    assert(std::is_same<T, double>() || std::is_same<T, float>());
```
Comment (mrexodia) [inline src/dbg/stringformat.cpp:26]: This should be static_assert

Comment (torusrxxx) [reply src/dbg/stringformat.cpp]: There seems to be a problem with static_assert but I don't use VS2013 anymore.

**Comment by mrexodia** [comment]
I’ll check it out, seems like a very strange error…

File: src/dbg/stringformat.cpp (hunk 1 above)
Comment (mrexodia) [reply src/dbg/stringformat.cpp]: I fixed it, had to use `std::is_same<T, double>::value`

**Comment by mrexodia** [comment]
Thanks!

**Pull request merged**